### ISSUE TITLE
The extension ms-ceintl.vscode-language-pack-zh-hans didn't work

### ISSUE THREAD

**Issue by tedyage** [gitpod-io/gitpod#4402]
### Bug description

The installed extension ms-ceintl.vscode-language-pack-zh-hans didn't work as it said in the manual.
![image](https://user-images.githubusercontent.com/32167109/120956051-58d71300-c785-11eb-9909-6c4887d2ce7e.png)

### Steps to reproduce

1. Open a workspace.
2. Search for extensions with keyword "Chinese", select one named "ms-ceintl.vscode-language-pack-zh-hans"  and install it on the workspace.  
3. Ctrl + Shift + p，and insert "display" as it mentioned in the manual.

### Expected behavior

Some options can list after insert display in the palette, and the Chinese option can be selected.

### Anything else?

Host OS version: centos 7.6
Gitpod version: 0.6.0_selfhosted
Theia version: 0.6.0

**Comment by akosyakov** [comment]
I don't think we properly load internalisation tables in the Gitpod Code Server yet. Maybe it is a reason.

**Comment by akosyakov** [comment]
We indeed does not load other locales in Gitpod Code Server yet, but VS Code web actually does not support localisation: https://github.com/microsoft/vscode/issues/82595

**Comment by akosyakov** [comment]
Although VS Code Web should detect your system locale and Gitpod Code Server then should provide proper translations. I will do a PR.

**Comment by akosyakov** [comment]
I have a fix which loads a proper translations for extensions: https://github.com/gitpod-io/vscode/commit/7762de62a5ff47d81475f1cbb5aac3e9cb683944 But since VS Code Web does not allow to switch locales but relies on browser locale, and use en locale for core code I think it is better to wait till these issues are resolved to provide the consistent experience.
